user/probe_bash.go

```diff
@@ -114,6 +115,12 @@ func (this *MBashProbe) setupManagers() {
 				//UprobeOffset: 0x8232, 	//若找不到 readline 函数，则使用offset便宜地址方式。
 				BinaryPath: binaryPath, // 可能是 /bin/bash 也可能是 readline.so的真实地址
 			},
+			{
+				Section:          "uretprobe/bash_retval",
+				EbpfFuncName:     "uretprobe_bash_retval",
+				AttachToFuncName: "execute_command",
```

Comment:
Can you give some source code about bash return?

Reply:
the bash workflow like this
```
 main()
        |
   reader_loop()       解析
        |--------------------------->read_command()-->parse_command()-->yyparse()-->yylex()-->read_token()-->read_token_word()
        |                                 |                               |                       |                 |
 execute_command() <-------------- current_command <--------------- global_command <------------token------------word
        |
execute_command_internal()
        |
 execute_xxxx_command()
        |
execute_simple_command()
        |
        |--->expand_words()-->expand_word_list_internal()
        |                                                                  子进程
        |------------------------------------->execute_disk_command()------------->shell_execve()-->execve()                
        |                  磁盘命令                       |                |                       |
        |函数及内置命令                              make_child()          |                       |FAILED
        |                                                |                |                       |
execute_builtin_or_function()                          fork()----------->pid                      ->execute_shell_script()
                                                                          |
                                                                          --------->return(result)
                                                                            父进程
```